from . import RCException


class FatalError(RCException):
    pass
